### ISSUE TITLE
Missing translations in settings of admin.0 instance

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.admin#998]
Admin 5.1.9

The following output in the browser console is shown if you enter in menu instances admin.0 settings:

```
Translate: Let's Encrypt SSL
 Translate: Access to instances
 Translate: Auto update
 Translate: Demo
 Translate: Discard
 Translate: Events threshold value
 Translate: IP
 Translate: Let's Encrypt SSL
 Translate: Login background
 Translate: Run as
 Translate: Secure(HTTPS)
 Translate: Use old UI
 Translate: every 12 hours
 Translate: every 2 days
 Translate: every 2 weeks
 Translate: every 3 days
 Translate: every day
 Translate: every week
 Translate: manually
 Translate: monthly
```

**Comment by Feuer-sturm** [comment]
@GermanBluefox 
Admin 5.1.10

Mentioned output "Translate: ...." is fixed :-)

Now there are two warnining  if you enter in menu instances admin.0 settings:
![grafik](https://user-images.githubusercontent.com/25422320/125161222-6109e000-e181-11eb-90c9-c654d786a70c.png)

**Comment by Feuer-sturm** [comment]
@Apollon77 @GermanBluefox Is my last comment relevant for Admin5 or is it a translation topic in weblate? Warning is available in Admin 5.1.13 as well.

**Comment by Feuer-sturm** [comment]
Issue (Now there are two warnining if you enter in menu instances admin.0 settings:)  is still there in Admin v5.1.28
